src/signup/action-creators/signUpSubmitActions.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+'use strict'
+
+import { ProfileUtils } from '../type-methods'
+
+import * as client from '../../core/client'
+
+import type { Action } from '../../actions'
+
+export function submit () {
+  return (dispatch: any, getState: any) => {
```

Comment:
no. it is a redux-thunk thing. It injects dispatch and getState automatically in case an actionCreator returns a faction, not an object